bundles/framework/personaldata/PublishedMapsList.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import { Message, Confirm } from 'oskari-ui'
+import { Table } from 'oskari-ui/components/Table'
+import { DeleteOutlined, EditOutlined } from '@ant-design/icons'
+import { red } from '@ant-design/colors'
+import styled from 'styled-components';
+import { showHTMLrender } from './view/HTMLrender/HTMLrender'
```

Comment:
This might be easier to find/understand later on if it was something like:
```
import { showSnippetPopup } from './view/embedded/SnippetPopup';
```